modules/monitoring-configuring-shiftstack-scraping.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/shiftstack-prometheus-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="monitoring-configuring-shiftstack-scraping_{context}"]
+= Collecting cluster metrics from the federation endpoint
+
+You can open the federation endpoint of your {product-title} cluster to make metrics available to a {rhoso-first} cluster to practice pull-based monitoring.
```

Comment:
As far as I understand there is no need to open any federation endpoint, it's already opened so we can just query it. @pierreprinetti @mandre please correct me if I'm wrong.
```suggestion
You can make use of the federation endpoint of your {product-title} cluster to make metrics available to a {rhoso-first} cluster to practice pull-based monitoring.
```

Reply:
```suggestion
You can employ the federation endpoint of your {product-title} cluster to make metrics available to a {rhoso-first} cluster to practice pull-based monitoring.
```